src/platform/silabs/wifi/rs911x/ncp/rsi_wlan_config.h

```diff
@@ -46,18 +46,9 @@
 
 //! To set Extended custom feature select bit map
 #if WIFI_ENABLE_SECURITY_WPA3_TRANSITION
-#ifdef CHIP_9117
```

Comment:
are we sure this is to be removed? I can see the XTAL_CLK and RAM_MODES to be different, are we maintaining that in a different location?

Reply:
The file isn't included by any of the SiWx files at the Matter levels or by files in the Wifi SDK that we use so i think it is safe to remove.
From the blame, i think this was added in the early days of the 917 when it was still integrated to the 9116 APIs which it isn't anymore.